packages/app/src/components/nominationInfo/index.js

```diff
@@ -1,14 +1,23 @@
 import React, { useState, useContext, useEffect } from 'react';
 import { ActiveNominationContext } from '../../utils/context/ActiveNominationContext';
 import ApplicationDetail from "./applicationDetail";
+import HealthProviderDetail from "./healthProviderDetail";
 import styles from "./styles.module.css";
 
 
+
 function NominationInfo() {
   const [activeNomination, setActiveNomination] = useContext(
     ActiveNominationContext
   );
 
+
+  
+  const { hospitalCity, hospitalState, hospitalZipCode } = activeNomination;
```

Comment:
Nitpick: Let's add `admissionDate` to the variables we're destructuring here. That way we can remove this line:

```
const { admissionDate } = activeNomination;
```